src/app/Processor/ProcessorDetailPage/ProcessorDetailPage.tsx

```diff
@@ -121,33 +230,65 @@ const ProcessorDetailPage = (): JSX.Element => {
                 <Stack hasGutter={true}>
                   <StackItem>
                     <TextContent>
-                      <Text component="h1">{processor.name}</Text>
+                      <Text component="h1">{currentProcessor.name}</Text>
                     </TextContent>
                   </StackItem>
                   <StackItem>
-                    <StatusLabel status={processor.status ?? ""} />
+                    <StatusLabel status={currentProcessor.status ?? ""} />
                   </StackItem>
                 </Stack>
               </SplitItem>
-              <SplitItem>
-                <Dropdown
-                  onSelect={actionsSelect}
-                  toggle={
-                    <DropdownToggle
-                      id="toggle-id"
-                      onToggle={actionsToggle}
-                      toggleIndicator={CaretDownIcon}
-                    >
-                      {t("common.actions")}
-                    </DropdownToggle>
-                  }
-                  isOpen={isActionsOpen}
-                  dropdownItems={actionItems}
-                />
-              </SplitItem>
+              {!isEditing && (
```

Comment:
@vpellegrino the `ProcessorDetailPage.test.tsx` does not exist. Can you consider to create such file? There are several conditions which affect how the page will look like. I do not expexct that you will fix it right now.

Reply:
@sjamboro, it seems we don't have jest tests for pages. I expect them to be covered by IT.
At least, this is what we agreed til now.
Please, @kelvah, correct me if I'm wrong.

Reply:
That's correct @vpellegrino